src/index.rst

```diff
@@ -612,10 +643,67 @@ Change *LSST_VERSION* in the example yaml accordingly: ::
      inCollection: "HSC/RC2/defaults"
      dataQuery: "exposure = 34342 AND detector = 10"
 
+For different ``butlerConfig`` directory, you also need to grant group permission for PanDA to access the butler::
+
+   $> chmod g+rws /sdf/group/rubin/repo/main/u/<your_operator_name>
+
 Now ready to submit the workflow: ::
 
    $> bps submit test_sdf.yaml
 
+Submit a development workflow to USDF
+-------------------------------------
+
+To submit a development workflow to S3DF, please at first check `Submit a workflow to USDF`_.
+Here we only list the differences.
+
+Copy the environment setup script from cvmfs to your local directory and update the lsst
+setup part to your private repo: ::
+
+   $> latest=$(ls -td /cvmfs/sw.lsst.eu/linux-x86_64/panda_env/v* | head -1)
+   $> cp $latest/setup_panda_s3df.sh .
+   $> <update setup_panda_s3df.sh>
+   $> source ./setup_panda_s3df.sh
+
+``Note``: Make sure PanDA can read your private repo: ::
+
+   $> chmod g+rxs <your private development repo>
+
+For the submission yaml file ``test_usdf.yaml``, you need to change the ``runnercommand`` to point
```

Comment:
As a general comment, when writing documents that are tracked in git, we try to do a "one sentence per line" approach so that git diff becomes manageable and will show an entire sentence that has changed.